.github/PULL_REQUEST_TEMPLATE/analyst_experience_team.md

```diff
@@ -0,0 +1,48 @@
+## Summary
+
+Summarize your PR. If it involves visual changes include a screenshot or gif.
+
+- **Related issue:** < LINK_TO_RELATED_ISSUE >
+- **Feature flag:** < FEATURE_FLAG_HERE >
+- **Docs ticket:** < LINK_TO_DOCS_TICKET_HERE >
+- **Schema changes:** < LINK_SCHEMA_CHANGE_TICKET_HERE >
+
+### How to test
```

Comment:
It would be great if we can link the test plan as well so people could have even more context about how the functionality works and what to expect.